core/vm/analysis.go

```diff
@@ -79,6 +79,13 @@ func codeBitmapInternal(code, bits bitvec) bitvec {
 		if int8(op) < int8(PUSH1) { // If not PUSH (the int8(op) > int(PUSH32) is always false).
```

Comment:
This condition needs to be moved below, otherwise condition in line 84 is never reached. (0x5c/0x5d < 0x60)